tinystatus

```diff
@@ -66,7 +66,7 @@ while IFS="$(printf '\n')" read -r line; do
 done < "${checkfile}"
 wait
 
-cat << EOF
+cat << EOF >> "${tmp}/report/index.html"
```

Comment:
Shouldn't this be `>` because we want to create a new file each time?

Reply:
The entire `$tmp` directory is later deleted [here](https://github.com/bderenzo/tinystatus/pull/8/files#diff-f05452a0e3a1e3c57603a770df4c8ad268e6f9ddcfa06bc007fc75e0d2201845L126).

Reply:
Ah! Oops, missed that. Thanks :)